README.md

```diff
@@ -101,12 +108,9 @@ Which of the following statements will work, assuming jQuery is loaded?
 
 Select all that apply:
 ```
-[] `$(".post").css("background", "peachpuff")`
-[] `$(".post").innerHTML`
-[] `$(".post").html()`
-[] `document.getElementsByClassName("post")[0].innerHTML`
-[] `document.getElementsByClassName("post").innerHTML`
-```
+[+] `$(".post").css("background", "peachpuff")`
+[+] `$(".post").html()`
+
```

Comment:
Close! It's sort of a trick question. `document.getElementsByClassName("post")[0].innerHTML` will also work, even though it isn't jQuery.